composer.json

```diff
@@ -19,16 +19,16 @@
         }
     ],
     "require": {
-        "php":                                  ">=5.3.9",
+        "php":                                  "~5.5",
         "a2lix/translation-form-bundle":        "~2.0",
         "doctrine/collections":                 "~1.2",
-        "doctrine/common":                      "~2.3",
+        "doctrine/common":                      "~2.3,<2.5",
```

Comment:
Version `2.5.0` has security issue and version `2.5.1` fixes that issue but introduces some bugs that affects our `@products` suite.

I think we may set `2.5` as default in `0.16`, but it'll need more work.